packages/cloud-cognitive/src/global/js/utils/props-helper.js

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright IBM Corp. 2020, 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// helpers functions for component props
+
+// remove undesired props from component props object
+export const stripUnwantedProps = (props, blacklist) => {
```

Comment:
@dcwarwick @davidmenendez  FYI 526 introduces a second pattern for stripping props, we should probably pick one and go with it.
This is from actionBarItem
const reservedProps = [
  'hasIconOnly',
  'kind',
  'size',
  'tooltipPosition',
  'tooltipAlignment',
  'type',
];
// Base props on Carbon Button
const propTypes = { ...Button.propTypes };
const defaultProps = { ...Button.defaultProps };
// Remove reserved props
reservedProps.forEach((prop) => {
  delete propTypes[prop];
  delete defaultProps[prop];
});